packages/restapi/src/lib/spaceV2/inviteToJoin.ts

```diff
@@ -0,0 +1,191 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import * as Peer from 'simple-peer';
+import { produce } from "immer";
+
+import { SpaceV2 } from "./SpaceV2";
+import { SpaceInviteInputOptions } from "./types";
+
+import getIncomingIndexFromAddress from "../video/helpers/getIncomingIndexFromAddress";
+import { VideoCallStatus } from "../types";
+import { getIceServerConfig } from "../video/helpers/getIceServerConfig";
+import sendSpaceNotification from './helpers/sendSpaceNotification';
+import { VIDEO_CALL_TYPE } from '../payloads/constants';
+import isJSON from '../video/helpers/isJSON';
```

Comment:
lets separate video imports and add a comment for future references. may confuse someone why are we moving some helpers to spaceV2 and using rest from video